### PR TITLE
feat: allow setting currency in the CoA importer

### PR DESCRIPTION
**Pull request by Alchez**
Required for https://github.com/ParsimonyGit/parsimony/pull/93.

### CODE REVIEW

**Comment by vjFaLk** [comment]
Task linked: [CU-1f96ht Update Chart of Accounts to handle multiple marketplaces](https://app.clickup.com/t/1f96ht)

**Comment by Alchez** [comment]
Made upstream to [develop](https://github.com/frappe/erpnext/pull/23687)

File: erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py [hunk 1]
```diff
@@ -158,36 +163,47 @@ def return_parent(data, child):
...
+	charts_map = defaultdict(dict)
+	paths = []
 
 	line_no = 3
 	error_messages = []
 
-	for i in data:
-		account_name, dummy, account_number, is_group, account_type, root_type = i
+	for row in data:
+		account_name, parent_account, account_number, is_group, account_type, root_type, *others = row
```
Comment (vjFaLk) [inline erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py:173]: This is failing on python2.7 and erroring it out

File: erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py [hunk 2]
```diff
@@ -1,66 +1,72 @@
 # Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
```
Comment (vjFaLk) [inline erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py:1]: Why are there so many changes?! The formatting changes are making it impossible to see the actual changes. Either don't format, or move formatting to a separate commit.

**Review by vjFaLk — CHANGES_REQUESTED** [review]
https://travis-ci.com/github/ParsimonyGit/erpnext/jobs/460969313

There's a test related to COA that's failing.

File: erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py (hunk 2 above)
Comment (Alchez) [reply erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py:1]: There are a few other changes that I had to do to avoid a bunch of duplication.

For the formatting, I can only suggest using [no whitespace](https://github.com/ParsimonyGit/erpnext/pull/5/files?w=1) 😅

File: erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py (hunk 1 above)
Comment (Alchez) [reply erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py:173]: Seems like that syntax is not supported on 2.7. I can change it, but are we supporting that version though?

Comment (vjFaLk) [reply erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py:173]: Ideally we should be dropping it, but I will also have to change Travis config to not run 2.7 tests.
